src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/BatchExecutionTest.java

```diff
@@ -568,6 +570,47 @@ public void testBatchStatementCancellation() throws Exception {
         }
     }
 
+    @Test
+    public void testExecuteBatchColumnCaseMatching() throws Exception {
+        // Insert Timestamp using prepared statement when useBulkCopyForBatchInsert=true
+        try (Connection con = DriverManager.getConnection(connectionString
+                + ";useBulkCopyForBatchInsert=true;sendTemporalDataTypesAsStringForBulkCopy=false;")) {
+            try (Statement statement = con.createStatement()) {
+                TestUtils.dropTableIfExists(caseSensitiveTable, statement);
+                String createSql = "CREATE TABLE" + caseSensitiveTable + " (c1 varchar(10))";
+                statement.execute(createSql);
+            }
+            try (PreparedStatement preparedStatement = con.prepareStatement("INSERT INTO " + caseSensitiveTable + "(c1) VALUES(?)")) {
+                preparedStatement.setObject(1, "value1");
+                preparedStatement.addBatch();
+                preparedStatement.setObject(1, "value2");
+                preparedStatement.addBatch();
+                preparedStatement.executeBatch();
+            }
+        }
+    }
```

Comment:
this test case seems unnecessary as this is done everywhere so it's already tested many times. The only thing we need is the negative test when there's a case mismatch